index.tsx

```
@@ -32,37 +37,115 @@ import { Message } from "discord-types/general";
+
+    // Place right after the apps dropdown
+    children[4]?.props.children.splice(4, 0,
+        <Menu.MenuItem
+            id="pk-reproxy"
+            label={
+                <div className="reproxy">
+                    <div className="reproxy">Reproxy As...</div>
+                </div>
+            }
+            listClassName="pk-reproxy-list"
+            children={proxyMenuItems}
```

Biome

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)